### DISCUSSION TITLE
Audio level

### DISCUSSION THREAD

**Discussion by radiopiraat** [AzuraCast/AzuraCast#6619] [General]
I'm using the master_me processing and it work fine.
I have only one issue, i can only adjust the soundlevel to -2db
I should want it on 0 db this is also i use the level for the live djs.

**Comment by MajesticUniverse** [comment]
I didn't like "master_me", I tried all the modes. It seemed to me that it was very quiet for a music radio and that's why I used "Basic normalization and compression" - this mode satisfied me, the volume is right, and the sound doesn't press like "master_me", but I haven't tested Stereo Tools yet.
